### ISSUE TITLE
same syn_id present in the graph multiple times

### ISSUE THREAD

**Issue by a-maiti** [waitwaitforget/ImageNet-Hierarchy-Visualization#2]
Hi, 
Thanks for the tree. How is the hierarchy of the tree decided? Asking because labels are occurring multiple times in a tree.

**Comment by jS5t3r** [comment]
yes, that could be. https://observablehq.com/@mbostock/imagenet-hierarchy
